### PR TITLE
Article list auth

### PR DESCRIPTION
**Pull request by Blazexsam27**
Added article list associated for a particular user along with edit, delete functionality.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ gem "stimulus-rails"
 gem "jbuilder"
 
 # Use Redis adapter to run Action Cable in production
-# gem "redis", "~> 4.0"
+gem "redis", "~> 4.1.3"
```
Comment (suyashagr11) [inline Gemfile:31]: Why we need this gem?

Comment (suyashagr11) [inline Gemfile:31]: Why we need this gem? Please explain what this does, thanks!

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+        @comment = Comment.new()
+        @article = Article.find(params[:article_id])
+    end
+
     def create
         @article = Article.find(params[:article_id])
         @comment = @article.comments.create(comment_params)
-        redirect_to article_path(@article)
+        @comment.save
+        redirect_to user_article_path(current_user.id, @article.id)
```
Comment (suyashagr11) [inline app/controllers/comments_controller.rb:12]: Error handling missing in this action

File: db/migrate/20220126062422_create_articles.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@ def change
     create_table :articles do |t|
       t.string :title
       t.text :body
+      t.references :user, null: false, foreign_key: true
```
Comment (suyashagr11) [inline db/migrate/20220126062422_create_articles.rb:6]: Do you think `null: false` is required?
I guess adding reference with foreign by defaults adds this :)
Just have a look and confirm

**Review by suyashagr11 — CHANGES_REQUESTED** [review]
Good job @Blazexsam27 , just few questions I left in the comments.
Other than that few things:
1. Mentioning `account details` in PR desc would not help anyone as it of your local db
2. Whenever we have changes that impacts UI, you can add a demo video in the PR (look for chrome extension `loom` and that demo here)

Overall - Very good

File: Gemfile (hunk 1 above)
Comment (Blazexsam27) [reply Gemfile:31]: This one was for action cable purpose(during production) , i will remove it.

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (Blazexsam27) [reply app/controllers/comments_controller.rb:12]: I will fix it. Thanks.

File: db/migrate/20220126062422_create_articles.rb (hunk 3 above)
Comment (Blazexsam27) [reply db/migrate/20220126062422_create_articles.rb:6]: yes I was getting some issues while migration when I added this references column. So I regenerated the model using generate command, that is why these properties are auto-filled.

**Review by suyashagr11 — APPROVED** [review]

**Pull request merged**